README.md

```diff
@@ -1,17 +1,44 @@
-# Google CloudEvents – Python
+# Google Events Library for Python
 
-This repository contains types for CloudEvents issued by Google.
+This library provides classes of common event types on Google Cloud Platform
+(GCP). At this moment the following types are available:
 
-## Prerequisites
+| Event Type  | Python Class | Description |
+| ------------- | ------------- | ------------- |
+| Cloud Pub/Sub Message Published Event  | MessagePublishedEvent | This event is triggered when a new Cloud Pub/Sub event is published to a topic. |
+| Cloud Audit Log Written Event  | AuditLogWrittenEvent | The event is triggered when a new Cloud Audit Log entry is written.  |
 
-- Python 3.5 (or higher)
+## Installation and Usage
 
-## Generate
+**Note**: This library requires Python 3.7+.
 
-Generate libraries:
+To install this package, run
 
-```sh
-./gen.sh
+``` sh
+pip install --upgrade google-events
+```
+
+To use an event class, see the snippet below:
+
+``` python
+from google.events import MessagePublishedEvent
```

Comment:
importing `MessagePublishedEvent` directly from `google.events` makes an assumption that data messages are unique within that scope.

I think we want to see:

```py
from google.cloud.pubsub.topic.v1.messagePublished import MessagePublishedData
```

like seen with the index:

https://github.com/googleapis/google-cloudevents

Reply:
This is one of the concerns I have with the spec organization, as at this moment it is a little bit difficult to discover for developers. Shall we discuss this further in the next meeting?

Reply:
I think we want the full `Event type` The name `MessagePublishedEvent` is not guaranteed to be globally unique within `google.events`.

Ping me anytime for a meeting or chat, whatever works.